src/site/content/en/blog/squoosh-v2/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Announcing Squoosh v2
+subhead: New codecs support, updatd design, and CLI support!
+description: |
+  Major version change to Squoosh.app with new CLI feature.
```

Comment:
```suggestion
  New codecs support, updated design, and CLI support!
```
It's OK for the `description` to duplicate the `subhead` and in this case I think the `subhead` is more useful